src/components/ItemEditorPage/LeftPanel/Items/Items.tsx

```diff
@@ -105,50 +81,70 @@ export default class Items extends React.PureComponent<Props, State> {
   }
 
   handleTabChange = (targetTab: ItemPanelTabs) => {
-    const { onLoadPage } = this.props
-    const { currentPages } = this.state
+    const { onLoadPage, onResetReviewedItems } = this.props
+    const { currentPages, currentTab } = this.state
+
     if (targetTab === ItemPanelTabs.ALL_ITEMS) {
       onLoadPage(currentPages[ItemPanelTabs.ALL_ITEMS])
     }
+
+    if (targetTab === ItemPanelTabs.ALL_ITEMS) {
+      this.setState({ showAllItemsTabChangeModal: false }, () => onLoadPage(currentPages[ItemPanelTabs.ALL_ITEMS]))
```

Comment:
does this line replace the previous condition at line 87?

```
if (targetTab === ItemPanelTabs.ALL_ITEMS) {
  onLoadPage(currentPages[ItemPanelTabs.ALL_ITEMS])
}
```

Reply:
It does! I'll remove the older condition.